processor/src/main/java/org/mapstruct/ap/internal/model/source/SourceMethod.java

```diff
@@ -381,9 +381,7 @@ public boolean isMapMapping() {
 
     public boolean isEnumMapping() {
         if ( isEnumMapping == null ) {
-            isEnumMapping = getSourceParameters().size() == 1
-                && first( getSourceParameters() ).getType().isEnumType()
-                && getResultType().isEnumType();
+            isEnumMapping = MappingMethodUtils.isEnumMapping( this );
```

Comment:
Because I also need it in the `AbstractBaseBuilder` line 61. In the method `createForgedBeanAssignment`, which I should probable rename to `createForgedAssignment` now. 

I would have extracted the method to a `default` on the `Method`, but no Java 8 for us yet :).

What is your suggestion for reusing the logic?